packages/image_picker/image_picker_ios/example/ios/RunnerTests/PickerSaveImageToPathOperationTests.m

```diff
@@ -55,6 +55,76 @@ - (void)testSaveGIFImage API_AVAILABLE(ios(14)) {
   [self verifySavingImageWithPickerResult:result fullMetadata:YES];
 }
 
+- (void)testSaveBMPImage API_AVAILABLE(ios(14)) {
+  NSURL *imageURL = [[NSBundle bundleForClass:[self class]] URLForResource:@"bmpImage"
```

Comment:
This PR should fix https://github.com/flutter/flutter/issues/95885?

Reply:
Yeah it should, I'll add it to the fix list.